README.md

```diff
@@ -5,25 +5,25 @@ A educational Python demonstration of Threading and Mutex usage in a ticket sell
 
 #The problem
 
-This system simulates sales of Air Plane Tickets. In this simulation, there are:
+This system simulates sales of Airplane Tickets. In this simulation, there are:
 - 4 companies that fly the same route, with the same plane and the same amount of seats.
 - 2 Flight Agencies that sell the tickets to the clients
 - a certain amount of clients
 
 ### Flight Company
 Each flight company has:
 - 5 seats per plane
-- a total dispenses of 1000.00 'coins'
-- a Crazy manager that increases the price of the ticket in 10% for each ticket sold.
+- a total dispenses of 1000.00 'coins'   # It's not clear to me what this means - is it 'total budget'? Hafgan
```

Comment:
This was supposed to be what the company will spend for every flight ( fuel, services, taxes and ect).
Can you make it more clear ?
Thanks for the rest !